prashant/HTML/Assignment3/mean.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>MI-MEAN</title>
+
+        <style>
+            #heading{
+                text-align: center;
+            }
+            .nav_bar{
+                text-align: center;
+                margin-bottom: 50px;
+            }
+            
+            body{
+                background-color: coral;
+            }
+        </style>
+    </head>
+    <body>
+        <h1 id="heading" >Ambersoft</h1>
+        <div   class="nav_bar">
+            <a href="/home/prashantsahu/MI-11-MEAN/prashant/HTML/Assignment3/home.html" >Home</a> | 
+            <a href="/home/prashantsahu/MI-11-MEAN/prashant/HTML/Assignment3/mean.html">MI-MEAN</a> | 
+            <a href="/home/prashantsahu/MI-11-MEAN/prashant/HTML/Assignment3/contact.html">Contact</a>
```

Comment:
You have give absolute path to href so this will only work on your machine. To work for others give relative path .